tests/base/system_signal004/HandleManySignals.idr

```diff
@@ -11,8 +11,8 @@ main = do
   [Right (), Right (), Right ()] <- sequence $ replicate 3 (raiseSignal SigABRT)
     | _ => putStrLn $ "got non-zero exit from a system call"
   sleep 1
-  [SigABRT, SigABRT, SigABRT] <- handleManyCollectedSignals (limit 4)
-    | (_ :: _ :: _ :: [])  => putStrLn "received the wrong signals."
+  [SigABRT] <- handleManyCollectedSignals (limit 4)
```

Comment:
Can we change this test case to raise and collect two different types of signals so that the test still exercises the fact that `handleManyCollectedSignals` will return all available signals (up to its limit argument)?

`SIGILL` is available on both POSIX and Windows platforms so that might be a good second signal for the test case even though it is an unlikely candidate in end-user code.

Reply:
Done.